tests/helpers/test_template.py

```diff
@@ -1645,6 +1645,26 @@ def test_base64_decode(hass: HomeAssistant) -> None:
     )
 
 
+def test_base64_decode_raw(hass: HomeAssistant) -> None:
+    """Test the base64_decode with encoding filter."""
+    assert (
+        template.Template(
+            '{{ "aG9tZWFzc2lzdGFudA==" | base64_decode(None) }}', hass
+        ).async_render()
+        == b"homeassistant"
+    )
+
+
+def test_base64_decode_encoding(hass: HomeAssistant) -> None:
+    """Test the base64_decode with encoding filter."""
+    assert (
+        template.Template(
+            '{{ "aG9tZWFzc2lzdGFudA==" | base64_decode("ascii") }}', hass
+        ).async_render()
+        == "homeassistant"
+    )
```

Comment:
Please add the two asserts to the test `test_base64_decode`

Reply:
Ok, done